### PR TITLE
[quant] Rename "2.0" to "2", remove "static"

### PR DESCRIPTION
**Pull request by andrewor14**
Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Fixes #ISSUE_NUMBER

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

File: prototype_source/prototype_index.rst [hunk 1]
```diff
@@ -207,7 +207,7 @@ Prototype features are not available as part of binary distributions like PyPI o
    prototype/fx_graph_mode_ptq_dynamic.html
    prototype/fx_graph_mode_ptq_static.html
    prototype/graph_mode_dynamic_bert_tutorial.html
-   prototype/quantization_in_pytorch_2_0_export_tutorial.html
+   prototype/quantization_in_pytorch_2_export_tutorial.html
```
Comment (andrewor14) [inline prototype_source/prototype_index.rst:210]: @jerryzh168 which file is this referring to? I couldn't find it

Comment (jerryzh168) [reply prototype_source/prototype_index.rst:210]: this is the pt2e_quantizer file

File: prototype_source/pt2e_quantizer.rst [hunk 2]
```diff
@@ -307,7 +307,7 @@ functions that are used in the example:
 Conclusion
 ^^^^^^^^^^^^^^^^^^^
 
-With this tutorial, we introduce the new quantization path in PyTorch 2.0. Users can learn about
-how to define a ``BackendQuantizer`` with the ``QuantizationAnnotation API`` and integrate it into the quantization 2.0 flow.
+With this tutorial, we introduce the new quantization path in PyTorch 2. Users can learn about
+how to define a ``BackendQuantizer`` with the ``QuantizationAnnotation API`` and integrate it into the quantization 2 flow.
```
Comment (jerryzh168) [inline prototype_source/pt2e_quantizer.rst:311]:
```suggestion
how to define a ``BackendQuantizer`` with the ``QuantizationAnnotation API`` and integrate it into the PyTorch 2 export quantization flow.
```

File: prototype_source/pt2e_quantizer.rst [hunk 3]
```diff
@@ -307,7 +307,7 @@ functions that are used in the example:
...
 Examples of ``QuantizationSpec``, ``SharedQuantizationSpec``, ``FixedQParamsQuantizationSpec``, and ``DerivedQuantizationSpec``
-are given for specific annotation use case. This is a prerequisite to be able to quantize a model in PyTorch 2.0 Export Quantization flow. You can use `XNNPACKQuantizer <https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/quantizer/xnnpack_quantizer.py>`_ as an example to start implementing your own ``Quantizer``. After that please follow `this tutorial <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq_static.html>`_ to actually quantize your model.
+are given for specific annotation use case. This is a prerequisite to be able to quantize a model in PyTorch 2 Export Quantization flow. You can use `XNNPACKQuantizer <https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/quantizer/xnnpack_quantizer.py>`_ as an example to start implementing your own ``Quantizer``. After that please follow `this tutorial <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq.html>`_ to actually quantize your model.
```
Comment (jerryzh168) [inline prototype_source/pt2e_quantizer.rst:313]:
```suggestion
are given for specific annotation use case. You can use `XNNPACKQuantizer <https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/quantizer/xnnpack_quantizer.py>`_ as an example to start implementing your own ``Quantizer``. After that please follow `this tutorial <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq.html>`_ to actually quantize your model.
```

File: prototype_source/pt2e_quantizer.rst [hunk 4]
```diff
@@ -1,9 +1,9 @@
-How to Write a ``Quantizer`` for PyTorch 2.0 Export Quantization
+How to Write a ``Quantizer`` for PyTorch 2 Export Quantization
 ================================================================
 
 **Author**: `Leslie Fang <https://github.com/leslie-fang-intel>`_, `Weiwen Xia <https://github.com/Xia-Weiwen>`__, `Jiong Gong <https://github.com/jgong5>`__, `Kimish Patel <https://github.com/kimishpatel>`__, `Jerry Zhang <https://github.com/jerryzh168>`__
 
-.. note:: Quantization in PyTorch 2.0 export is still a work in progress.
+.. note:: Quantization in PyTorch 2 export is still a work in progress.
```
Comment (jerryzh168) [inline prototype_source/pt2e_quantizer.rst:6]: maybe we can remove this

**Review by jerryzh168 — APPROVED** [review]
LG, thanks! some comments inline

File: prototype_source/prototype_index.rst (hunk 1 above)
Comment (jerryzh168) [reply prototype_source/prototype_index.rst:210]: could you add the other two files to this list as well?

**Review by svekars — APPROVED** [review]

File: prototype_source/prototype_index.rst [hunk 5]
```diff
@@ -207,7 +207,9 @@ Prototype features are not available as part of binary distributions like PyPI o
...
```
Comment (jerryzh168) [inline prototype_source/prototype_index.rst:210]: this needs to be updated as well, to pt2e_quantizer.html

Comment (andrewor14) [reply prototype_source/prototype_index.rst:210]: oh I see. I misunderstood your previous comment

File: prototype_source/pt2e_quant_ptq.rst [hunk 6]
```diff
@@ -1,4 +1,4 @@
-(prototype) PyTorch 2.0 Export Post Training Static Quantization
+(prototype) PyTorch 2 Export Post Training Static Quantization
```
Comment (leslie-fang-intel) [inline prototype_source/pt2e_quant_ptq.rst:1]: @jerryzh168 do we need to remove the `Static` in the title?

Comment (jerryzh168) [reply prototype_source/pt2e_quant_ptq.rst:1]: yes I think so, @andrewor14 could you update the PR to remove static in the doc as well

**Pull request merged**